src/lib/hokusai.py

```diff
@@ -3,8 +3,7 @@
 from lib.util import run_cmd
 
 
-def env_unset(project_dir, env, var_names):
-  ''' run hokusai env unset for vars for project '''
-  for var in var_names:
-    logging.debug(f'unsetting {var}...')
-    run_cmd(f'hokusai {env} env unset {var}', project_dir)
+def env_unset(project_dir, env, var):
```

Comment:
operate on 1 var instead

Reply:
yes I like this more as only taking one var!